### PR TITLE
Added all the things to Other Tools page

### PR DESCRIPTION
**Pull request by justaddcoffee**
Added blurbs for Phenogrid, Dipper, HPO Case Annotator, and BOOM, have a look [here](https://justaddcoffee.github.io/monarch-ui/tools/other-tools). 

@iimpulse let me know if you have a better link for phenogrid
@cmungall I pieced together a brief summary for BOOM/kBOOM, feel free to change

### CODE REVIEW

**Comment by kshefchek** [comment]
is there an advantage to using html instead of markdown to author these pages? curious if something is not supported in the vue-markdown-loader lib

**Comment by justaddcoffee** [comment]
> is there an advantage to using html instead of markdown to author these pages? curious if something is not supported in the vue-markdown-loader lib

Not sure, I was just using HTML because that was what was there before - should we convert this page to markdown?

**Comment by kshefchek** [comment]
if it's expressive and flexible enough, I advocate for using markdown on any page that is purely content, but I haven't written any of these pages so I'm not aware of the caveats

**Comment by monicacecilia** [comment]
@justaddcoffee - I am less inclined to make changes on this right now, because I fear we'll go down the rabbit hole of updating ALL of the pages. I say, if it is not affecting anything in a terribly devastating way (e.g., the page looks terrible!), then let's merge this content right now and make a note of updating ALL static pages to md after v1.0 is out the door. 
Does that sound ok to you, @kshefchek?

File: src/views/OtherTools.md [hunk 1]
```diff
@@ -74,7 +74,42 @@
...
+    </div>
+    <div class="card">
+            <h4>HPO Case Annotator</h4>
+            <p>
+            <a href="https://github.com/monarch-initiative/HpoCaseAnnotator">The HPO Case Annotator (HCA)</a> GUI is a
+            Java app created with aim to make the biocuration of published pathogenic human variants easier.
+            </p>
+    </div>
+    <div class="card">
+            <h4>BOOM: Bayes OWL Ontology Merging</h4>
```
Comment (monicacecilia) [inline src/views/OtherTools.md]: Dear @justaddcoffee, please don't hate me, but after saying yay to this bit, I'm now having second thoughts about k-boom. So, this being a semi-automatic method that was initially used to construct Mondo, makes me feel like we are 'too down in the weeds' for what seems to be a more generic 'Monarch Tools' page. 
If we're doing this, should we also be talking about ODK? 
..... I may be splitting hairs. What do you think, Justin? 
And, @cmungall - should we keep or axe for now?

**Comment by kshefchek** [comment]
html is totally fine, and it shouldn't block merging this

File: src/views/OtherTools.md (hunk 1 above)
Comment (justaddcoffee) [reply src/views/OtherTools.md]: Okay with me to remove k-boom

**Pull request merged**